smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/RuntimeConfigGenerator.java

```diff
@@ -197,11 +197,20 @@ void generate(LanguageTarget target) {
                 }
                 int indentation = target.equals(LanguageTarget.SHARED) ? 1 : 2;
                 configs.forEach((key, value) -> {
-                    writer.indent(indentation).disableNewlines().openBlock("$1L: config?.$1L ?? ", ",\n", key,
-                        () -> {
-                            value.accept(writer);
-                        });
-                    writer.dedent(indentation);
+                    String defaultPrefix = "config?.$1L ?? ";
+                    if (key.equals("requestHandler")) {
+                        defaultPrefix = "";
+                    }
+                    writer
+                        .indent(indentation)
+                        .disableNewlines()
+                        .openBlock(
+                            "$1L: " + defaultPrefix, ",\n", key,
+                            () -> {
+                                value.accept(writer);
+                            }
+                        )
+                        .dedent(indentation);
```

Comment:
This type of change probably means that the existing `TypeScriptIntegration` hook is not extensible enough, as special casing `requestHandler` doesn't seem right.

Reply:
The current type of `Map<String, Consumer<TypeScriptWriter>>` doesn't have any room to indicate whether the default `config?.someProperty ??` should be prefixed or not. 

What do you suggest?

Reply:
make a `Set<String>` static collection to indicate which fields should not have a default prefix of the value defined on config?

Reply:
going to add a method to TypeScriptIntegration

Reply:
Decided on keeping a static map of default prefixes rather than exposing a new hook in `TypeScriptIntegration`.